tests/suites/0_stateless/20+_others/20_0012_privilege_access.result

```diff
@@ -1,34 +1,34 @@
 1
 test -- select
-ERROR 1105 (HY000) at line 1: PermissionDenied. Code: 1063, Text = Permission denied, privilege [Select] is required on 'default'.'default'.'t20_0012_a' for user 'test-user'@'%' with role public.
+ERROR 1105 (HY000) at line 1: PermissionDenied. Code: 1063, Text = Permission denied, privilege [Select] is required on 'default'.'default'.'t20_0012_a' for user 'test-user'@'%' with roles [public].
 1
-ERROR 1105 (HY000) at line 1: PermissionDenied. Code: 1063, Text = Permission denied, privilege [Select] is required on 'default'.'default'.'t20_0012_b' for user 'test-user'@'%' with role public.
+ERROR 1105 (HY000) at line 1: PermissionDenied. Code: 1063, Text = Permission denied, privilege [Select] is required on 'default'.'default'.'t20_0012_b' for user 'test-user'@'%' with roles [public].
 1
-ERROR 1105 (HY000) at line 1: PermissionDenied. Code: 1063, Text = Permission denied, privilege [Select] is required on 'default'.'default2'.'v_t20_0012' for user 'test-user'@'%' with role public.
+ERROR 1105 (HY000) at line 1: PermissionDenied. Code: 1063, Text = Permission denied, privilege [Select] is required on 'default'.'default2'.'v_t20_0012' for user 'test-user'@'%' with roles [public].
 1
 test -- clustering_information
-ERROR 1105 (HY000) at line 1: PermissionDenied. Code: 1063, Text = Permission denied, privilege [Select] is required on 'default'.'system'.'clustering_information' for user 'test-user'@'%' with role public.
+ERROR 1105 (HY000) at line 1: PermissionDenied. Code: 1063, Text = Permission denied, privilege [Select] is required on 'default'.'system'.'clustering_information' for user 'test-user'@'%' with roles [public].
 1
-ERROR 1105 (HY000) at line 1: PermissionDenied. Code: 1063, Text = Permission denied, privilege [Select] is required on 'default'.'system'.'fuse_snapshot' for user 'test-user'@'%' with role public.
+ERROR 1105 (HY000) at line 1: PermissionDenied. Code: 1063, Text = Permission denied, privilege [Select] is required on 'default'.'system'.'fuse_snapshot' for user 'test-user'@'%' with roles [public].
 1
-ERROR 1105 (HY000) at line 1: PermissionDenied. Code: 1063, Text = Permission denied, privilege [Select] is required on 'default'.'system'.'fuse_segment' for user 'test-user'@'%' with role public.
+ERROR 1105 (HY000) at line 1: PermissionDenied. Code: 1063, Text = Permission denied, privilege [Select] is required on 'default'.'system'.'fuse_segment' for user 'test-user'@'%' with roles [public].
 1
-ERROR 1105 (HY000) at line 1: PermissionDenied. Code: 1063, Text = Permission denied, privilege [Select] is required on 'default'.'system'.'fuse_block' for user 'test-user'@'%' with role public.
+ERROR 1105 (HY000) at line 1: PermissionDenied. Code: 1063, Text = Permission denied, privilege [Select] is required on 'default'.'system'.'fuse_block' for user 'test-user'@'%' with roles [public].
```

Comment:
Better to add this test:

user foo has two or more roles: test_role1, teest_role2 ... which without select privilege.

And then execute select default'.'system'.'fuse_block check the privilege access.